hieradata/common.yaml

```diff
@@ -98,7 +98,18 @@ nginx::nginx_locations:
     location: '= /error/503.html'
     location_custom_cfg:
       - 'root /srv/mirror_data'
-
+  'query-string-assets':
+    vhost: 'assets-origin'
```

Comment:
I asked, and the answer was yes, although we hope nothing comes through to assets with a query string in the first place.